src/main/java/org/mule/tools/apikit/model/ScaffolderResource.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.apikit.model;
+
+public class ScaffolderResource {
+
+}
```

Comment:
????

Reply:
This class will be used with the munit scaffolder (which is not ready yet), it was defined in the "new Scaffolder api" document that all the mule configs should contain it.
Anyways I have some doubts about it, I will talk with Diego and Juani.